http-server/src/main/java/xtdb/jackson/JsonLdValueOrPersistentHashMapDeserializer.java

```diff
@@ -0,0 +1,72 @@
+package xtdb.jackson;
```

Comment:
Given this is significantly derived from Jsonista I'd add a notice to the top of this file with the Jsonista license - see `xtdb.backtick` for an example.